frontends/web/src/locales/en/app.json

```diff
@@ -380,6 +380,7 @@
     "next": "Next",
     "ok": "OK",
     "previous": "Previous",
+    "proceedOnBitBox": "Proceed on the BitBox02",
```

Comment:
I often thought that we should start saying "BitBox" instead of "BitBox02" to make integration of a potential BitBox03 etc. simpler in the future, so: "Proceed on your BitBox". WDYT?

Reply:
yes I  absolutely agree, but this issue exists in many places. I am changing to just "BitBox" for now..

Reply:
done